lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -0,0 +1,890 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c), Entrust Datacard Corporation, 2019
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ecs_certificate
+author:
+    - Chris Trufan (@ctrufan)
+version_added: 2.9
```

Comment:
```suggestion
version_added: '2.9'
```
It's better to always quote version numbers as strings, so you don't accidentally end up with 2.10 being equal to 2.1 :)